### PR TITLE
Disable top bar: FullscreenGame version

### PR DESCRIPTION
**Pull request by garbear**
As a follow-up to #12742, this change separates the FullscreenVideo window name from the FullscreenGame window name. As a result, the top bar no longer shows up during gameplay.

## Motivation and Context
This creates a new fullscreen game class where future rendering stuff can be put.

## How Has This Been Tested?
Tested on Windows, no top bar appears inside game.

## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Improvement (non-breaking change which improves existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: xbmc/video/windows/GUIWindowFullScreenDefines.h [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this Program; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+#pragma once
+
+#define BLUE_BAR                          0
```
Comment (da-anda) [inline xbmc/video/windows/GUIWindowFullScreenDefines.h]: I know this PR didn't introduce this define, but what is "blue bar" supposed to be? The header? Might be good to rename this define into something more descriptive and especially not related to any skin/design choice.

Comment (garbear) [reply xbmc/video/windows/GUIWindowFullScreenDefines.h]: How many skinners read source code? I completely agree that this should be documented, but I'm not sure the source is the right place for skinning stuff. From a programmer's side it doesn't really matter, just another anonymous variable in a sea of many.

**Comment by garbear** [comment]
jenkins build this please

**Comment by garbear** [comment]
jenkins build this please

**Comment by garbear** [comment]
Linux x64 failure is unrelated. Gonna squash and test once more locally before merging.

**Pull request merged**